start.js

```diff
@@ -0,0 +1,207 @@
+let backgroundUrl = ["url('background/space-g52a222904_1920.jpg')",
+ "url('background/space-g10131bb9e_1920.jpg')",
+ "url('background/stars-g5d44ea9a5_1920.jpg')"];
+
+let Color = [
+    "red", "orange", "yellow", "lime", "green", "blue", "aqua", "purple", "black", 
+    "gray", "aliceblue", "blueviolet", "brown", "navy"
+];
+
```

Comment:
backgroundUrl, Color는 `const`를 써도 될 거 같아서 한 번 코멘트 달아봅니다 :)
추가로, 변수명이 대문자로 시작하는 건 오타일 듯해서 한번 남겨봐요

```
변수 선언에는 기본적으로 const를 사용하고 let은 재할당이 필요한 경우에 한정해 사용하는 것이 좋다. 원시 값의 경우, 가급적 상수를 사용하는 편이 좋다. 그리고 객체를 재할당하는 경우는 생각보다 흔하지 않다. const 키워드를 사용하면 의도치 않은 재할당을 방지해 주기 때문에 보다 안전하다.

var와 let, 그리고 const는 다음처럼 사용하는 것을 추천한다.

- ES6를 사용한다면 var 키워드는 사용하지 않는다.
- 재할당이 필요한 경우에 한정해 let 키워드를 사용한다. 이때 변수의 스코프는 최대한 좁게 만든다.
변경이 발생하지 않는(재할당이 필요 없는 상수) 원시 값과 객체에는 const 키워드를 사용한다. const 키워드는 재할당을 금지하므로 var, let 보다 안전하다.

출처: https://poiemaweb.com/es6-block-scope
```



```suggestion
const backgroundUrl = ["url('background/space-g52a222904_1920.jpg')",
 "url('background/space-g10131bb9e_1920.jpg')",
 "url('background/stars-g5d44ea9a5_1920.jpg')"];

const color = [
    "red", "orange", "yellow", "lime", "green", "blue", "aqua", "purple", "black", 
    "gray", "aliceblue", "blueviolet", "brown", "navy"
];

```

Reply:
두 의견 다 좋은 것 같습니다. 말씀하신 대로 수정하도록 하겠습니다!